api/types/src/convert.rs

```diff
@@ -350,7 +356,13 @@ impl<'a, R: MoveResolverExt + ?Sized> MoveConverter<'a, R> {
         }
         let table_info = match self.db.get_table_info(handle) {
             Ok(ti) => ti,
-            Err(_) => return Ok(None), // if table item not found return None anyway to avoid crash
+            Err(_) => {
+                aptos_logger::error!(
+                    "Table info not found for handle {:?}, can't decode table item",
+                    handle
+                );
```

Comment:
it's not really an error for simulation.. so let's at least make it warning instead of error?

Reply:
Yea we can warn. Probably better to warn given that's likely the common case especially when we turn on internal indexer for our fullnodes.